collection_mutation.cc

```diff
@@ -188,7 +188,8 @@ compact_and_expire_result collection_mutation_description::compact_and_expire(co
     for (auto&& name_and_cell : cells) {
         atomic_cell& cell = name_and_cell.second;
         auto cannot_erase_cell = [&] {
-            return cell.deletion_time() >= gc_before || !can_gc(tombstone(cell.timestamp(), cell.deletion_time()));
+            // The collection cell tombstones are never shadowable
```

Comment:
As an outsider to the compaction logic this comment is not verbose enough for me. Why is it never shadowable? I'd presume because a shadowable tombstone shadows an entire row, not individual cells? Are collections special in any way?

Reply:
> As an outsider to the compaction logic this comment is not verbose enough for me. Why is it never shadowable? I'd presume because a shadowable tombstone shadows an entire row, not individual cells? Are collections special in any way?

collections are not special in any way, it's just that only row tombstones can be shadowable.  Other objects don't carry shadowable tombstones.